posix.mak

```
@@ -180,7 +180,7 @@ DRUNTIME_LATEST_DIR=$G/druntime-${LATEST}
 PHOBOS_LATEST_DIR=$G/phobos-${LATEST}
 
 # stable dub and dmd versions used to build dpl-docs
-STABLE_DMD_VER=2.085.0
+STABLE_DMD_VER=2.087.0
```

Use the point release to avoid theoretically hitting some fixed bugs:
```suggestion
STABLE_DMD_VER=2.087.1
```